### PR TITLE
1 export parameters

### LINKED ISSUE #1: Test current package and expand parameters to building VT/checkpoint
- [ ] Write some tests for the spack package
- [ ] Maybe use the build script instead of just invoking cmake?
- [ ] Export up configuration parameters in VT that are useful to spack configuration
- [ ] Unify the names DarmaVt vs Checkpoint of the packages

### PR DESCRIPTION
**Pull request by jstrzebonski**
Fixes #1 

---

TODO:
- [x] Explore possibility to use `sanity_check_is_file` and `sanity_check_is_dir`

### CODE REVIEW

File: packages/darma-vt/package.py [hunk 1]
```diff
@@ -58,17 +58,73 @@ class DarmaVt(CMakePackage):
...
+            description='Enable prioritization of work')
+    variant('diagnostics_enabled', values=("0", "1"), default="1",
+            description='Enable VT component diagnostics for performance analysis')
+    variant('diagnostics_runtime_enabled', values=("0", "1"), default="0",
+            description='Enable VT component diagnostics at runtime by default')
+    variant('priority_bits_per_level', values=int, default=3,
+            description='Number of bits per level of priority in envelope')
+    variant('unity_build_enabled', values=("0", "1"), default="0",
+            description='Build with Unity/Jumbo mode enabled (requires CMake >= 3.16)')
+    variant('fcontext_enabled', values=("0", "1"), default="1",
```
Comment (cz4rs) [inline packages/darma-vt/package.py:91]: the default value in the docs is wrong for this (should be `0`)
I have a fix for this waiting (https://github.com/DARMA-tasking/vt/pull/1288/commits/681cdfc9fad11313e686e898d84d545a2ccc2837)

**Review by JacobDomagala — APPROVED** [review]

File: packages/darma-vt/package.py (hunk 1 above)
Comment (jstrzebonski) [reply packages/darma-vt/package.py]: Done.

**Review by cz4rs — APPROVED** [review]
looks good to me :+1:

**Review by lifflander — APPROVED** [review]
Looks good

**Review by cz4rs — APPROVED** [review]

**Pull request merged**